Tests/ReactiveSwiftTests/UnidirectionalBindingSpec.swift

```diff
@@ -50,6 +50,22 @@ class UnidirectionalBindingSpec: QuickSpec {
 				expect(value) == 1
 			}
 
+			it("should not deadlock on the main thread even if the context was switched to a different queue") {
+				let queue = DispatchQueue(label: #file)
+
+				target = BindingTarget(on: UIScheduler(),
+				                       lifetime: lifetime,
+				                       setter: { value = $0 })
+
+				let property = MutableProperty(1)
+
+				queue.sync {
+					_ = target <~ property
+				}
+
+				expect(value).toEventually(equal(1))
```

Comment:
This was `expect(value) == 1` as the binding target was guaranteed to be synchronous before.